woocommerce/woocommerce-test-reports
# Collect daily stats

This is part 2 to the stats project.  This part collects the stats from the daily test runs and writes them to a Google Sheet.  The credentials are saved as Github secrets and it uses the Google API library to do the heavy lifting of interacting with Google Sheets.

To test:

- Check Check the data page https://lanej0.github.io/woocommerce-test-reports/data
- Kick off a daily run
- Check the data page https://lanej0.github.io/woocommerce-test-reports/data -- there should be a new data point added for today.  There may be multiple data points for a single day, but we can clean those up manually once testing is done.
- Note: the size of test suite is only measured once a month (on the first) so if it's not the first, you won't see that chart change.
